api/v1alpha1/hostedcluster_types.go

```diff
@@ -574,6 +574,16 @@ type IBMCloudPlatformSpec struct {
 
 // PowerVSPlatformSpec defines IBMCloud PowerVS specific settings for components
 type PowerVSPlatformSpec struct {
+	// ApiKey is used to authenticate IBMCloud
+	//
+	// +immutable
+	ApiKey string `json:"apiKey"`
```

Comment:
I don't see this is used anywhere in this change, can we remove this if not

Reply:
Removed